stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/da70d67ff662b85e72e1f203defbf5e3028d03cd/snapshots/cardano-1.19.1.yaml
```

Comment:
I'd prefer we commit this file to the repository:

* easier to view/grep the dependencies
* unbreaks `stack2cabal` (currently doesn't handle remote urls in `resolver`)
* more static than a url